fhir2dataset/api_caller.py

```diff
@@ -124,9 +125,7 @@ def __init__(
         self.main_resource_alias = main_resource_alias
         self.elements = elements
```

Comment:
Could you add a class dosctring to explain the attributes of the class such as self.elements?

Reply:
It's done